bundles/org.openhab.ui.homebuilder/web/src/AppForm.vue

```diff
@@ -0,0 +1,206 @@
+<template>
+  <div class='row'>
+    <div class='col-sm-6' id="form">
+      <vue-form-generator :schema='schema' :model='model' :options='formOptions'>
+      </vue-form-generator>
+      <pre class='language-json hidden-xs-up'><code>{{ model }}</code></pre>
+    </div>
+    <div class='col-sm-6'>
+      <affix id="preview" relative-element-selector="#form" :offset="{ top: 65, bottom: 0 }">
+        <collapse v-if="model && model.filesGenerated.includes('items')" :uid="'items'" :heading="'Items'" :content='model.itemsType === "text" ? generateItems(model) : generateItemsJson(model)'></collapse>
+
+        <collapse v-if="model && model.filesGenerated.includes('sitemap')" :uid="'sitemap'" :heading="'Sitemap'" :content='generateSitemap(model)'></collapse>
+
+        <collapse v-if="model && model.filesGenerated.includes('habpanel')" :uid="'habpanel'" :heading="'HABPanel Dashboard'" :content='generateDashboard(model)'></collapse>
+      </affix>
+    </div>
+  </div>
+</template>
+
+<style lang="scss" src="./scss/app.scss">
```

Comment:
dead code

Reply:
Note that `src` parameter is used here. The empty line between `style` tag though is a result of JS Beautifier's formatter being funky in `*.vue` files ;-)